planning/autoware_freespace_planning_algorithms/include/autoware/freespace_planning_algorithms/abstract_algorithm.hpp

```diff
@@ -84,6 +88,13 @@ struct VehicleShape
     max_steering(vehicle_info.max_steer_angle_rad),
     base2back(vehicle_info.rear_overhang_m + margin / 2.0)
   {
+    setMinMaxDimension();
+  }
+
+  void setMinMaxDimension()
+  {
+    min_dimension = std::min(0.5 * width, base2back);
+    max_dimension = std::hypot(length - base2back, 0.5 * width);
```

Comment:
[nits]
this is rare but some vehicle may have longer base2back than base2front.